pkg/app/piped/executor/kubernetes/kubernetes_test.go

```diff
@@ -290,6 +290,211 @@ spec:
 
 }
 
+func TestApplyManifests(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
```

Comment:
[nits] no longer need to call `ctrl.Finish()`🙌 

ref: https://pkg.go.dev/github.com/golang/mock/gomock#:~:text=be%20invoked%20once.-,New%20in%20go1.14%2B%2C%20if%20you%20are%20passing%20a%20*testing.T%20into%20NewController%20function%20you%20no%20longer%20need%20to%20call%20ctrl.Finish()%20in%20your%20test%20methods.,-func%20(*Controller)%20RecordCall